### DISCUSSION TITLE
Multiple Instances of Antares

### DISCUSSION THREAD

**Discussion by leonuetzel** [flandreas/antares#409] [Q&A] [answered]
Just a Question: Is it safe to open multiple Instances of Antares, like opening two separate Projects in two separate Instances to watch at both simultaneously?

**Comment by flandreas** [comment, marked as answer]
Well, I've never tried this, but it should be safe, as long as you don't edit and save the same files. Both instances will operate on the same data directory where circuits are stored. And when Antares is quit, it stores some preference settings into files in the same directory.

So, just running multiple projects in parallel should be fine, but I wouldn't save circuits when multiple instances are running. To be on the save side, I would recommend to first make a backup copy of your data directory.
